package.json

```diff
@@ -64,8 +65,13 @@
     ]
   },
   "devDependencies": {
+    "@babel/preset-env": "^7.16.0",
+    "@babel/preset-react": "^7.16.0",
+    "@mdx-js/loader": "^1.6.22",
+    "@types/mdx-js__react": "^1.5.5",
     "@types/react-lazyload": "^3.1.1",
     "@types/react-slick": "^0.23.7",
+    "babel-loader": "8.1.0",
```

Comment:
You may wonder why we need to install `babel-loader` since `react-script` already provides the module. The mdxjs's [docs](https://mdxjs.com/getting-started/create-react-app/) said in `@mdx/react` needs bael-loader for some reason. I have no choice.